examples/01_bars.html

```diff
@@ -48,8 +48,8 @@
 
 					var elem = { x: startValue, domElement: domElement };
 
-					var updateCallback = function() {
-						this.domElement.style.left = this.x + 'px';
+					var updateCallback = function(object) {
+						object.domElement.style.left = this.x + 'px';
```

Comment:
`this.x` needs to be changed to `object.x`.

Reply:
Fixed